### DISCUSSION TITLE
Unable to get this to work.

### DISCUSSION THREAD

**Discussion by JohnRamminger** [UpendoVentures/generator-upendodnn#302] [General]
Are there specific versions of node I need to use.  I have tried both the SPA template and the Webforms template and have gotten errors in both.

I am using node 20.10.0.  When using the webforms template I get the following error

C:\Users\Administrator\AppData\Roaming\nvm\v20.10.0\node_modules\generator-upendodnn\generators\common\src-webforms\packages.config:3
    1| <?xml version="1.0" encoding="utf-8"?>
    2| <packages>
 >> 3|   <package id="DotNetNuke.Core" version="<%= dnnBuildVersionShort %>" targetFramework="net472" />
    4|   <package id="DotNetNuke.Instrumentation" version="<%= dnnBuildVersionShort %>" targetFramework="net472" />
    5|   <package id="DotNetNuke.Web" version="<%= dnnBuildVersionShort %>" targetFramework="net472" />
    6|   <package id="DotNetNuke.Web.Client" version="<%= dnnBuildVersionShort %>" targetFramework="net472" />

dnnBuildVersionShort is not defined

When trying to use the SPA template it is complaining about not being able to find the follow file.

  Determining projects to restore...
C:\Program Files\dotnet\sdk\8.0.100\NuGet.targets(156,5): error : Unable to load the service index for source https://d
otnet.myget.org/F/dotnet-core/api/v3/index.json. [C:\code\test3\Modules\Test3\RammWare.Modules.Test3.csproj]
C:\Program Files\dotnet\sdk\8.0.100\NuGet.targets(156,5): error :   The SSL connection could not be established, see in
ner exception. [C:\code\test3\Modules\Test3\RammWare.Modules.Test3.csproj]
C:\Program Files\dotnet\sdk\8.0.100\NuGet.targets(156,5): error :   The remote certificate is invalid according to the
validation procedure: RemoteCertificateNameMismatch [C:\code\test3\Modules\Test3\RammWare.Modules.Test3.csproj]
Creating sln.
Running dotnet restore.
× An error occured while running upendodnn:spa#install
Error upendodnn

Command failed with exit code 1: dotnet restore RammWare.Modules.Test3.csproj

Thanks for you help.

**Comment by WillStrohl** [comment]
Can you please provide more details about your development environment?  Can you confirm that you've installed the prerequisites in the `README`?  As long as your development environment is setup properly and you've installed the prerequisites, everything should work smoothly.

**Reply by JohnRamminger** [reply]
I am using Server 2019 with the latest build of visual studio 2022.  Node 20.10.0 installed with nvm.  If Visual studio 2019 would work better I can try that.  I am running the command line in administrator mode.

What  is your setup and I can try and match that.


Thanks for your hard work and for helping.

John
